lib/src/arithmetic/floating_point/floating_point_sqrt_simple.dart

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2025 Intel Corporation
+// SPDX-License-Indentifier: BSD-3-Clause
+//
+// floating_point_sqrt.dart
+// An abstract base class defining the API for floating-point square root.
+//
+// 2025 March 4
+// Authors: James Farwell <james.c.farwell@intel.com>,
+//Stephen Weeks <stephen.weeks@intel.com>,
+//Curtis Anderson <curtis.anders@intel.com>
+
+import 'package:rohd/rohd.dart';
+import 'package:rohd_hcl/rohd_hcl.dart';
+
+/// An square root module for FloatingPoint values
```

Comment:
Dart convention is to end the first sentence with a period, then continue to describe.

It would be good to describe any limitations here.  For example, what is m.3 inside?  What does that imply in terms of what this unit can or cannot do.